src/test/java/com/uid2/shared/auth/AuthorizableStoreTest.java

```diff
@@ -0,0 +1,79 @@
+package com.uid2.shared.auth;
+
+import com.uid2.shared.secret.KeyHashResult;
+import com.uid2.shared.secret.KeyHasher;
+import org.junit.Assert;
```

Comment:
We should be using JUnit Jupiter's asserts

`import static org.junit.jupiter.api.Assertions.*;`